core/modules/field_ui/field_ui.theme.inc

```diff
@@ -158,7 +164,7 @@ function theme_field_ui_view_modes($variables) {
     else {
       $default_count++;
       $row_classes[] = 'uses-default';
-      $row['label'] .= '&nbsp;<small>(<em>' . t('Uses the Default display mode') . '</em>)</small>';
+      $row['description'] = '<em>' . t('Uses the Default display mode') . '</em>';
```

Comment:
If we wanted to be _technically_ accurate, we should say:

```suggestion
      $row['description'] = '<em>' . t('Uses the Default display mode settings') . '</em>';
```

Reply:
...or:

```suggestion
      $row['description'] = '<em>' . t('Uses the settings of the Default display mode') . '</em>';
```

Reply:
I feel this would be better addressed as part of https://github.com/backdrop/backdrop-issues/issues/4891#issuecomment-764165596 (comment) (comment added).